PGM/src/main/java/tc/oc/pgm/mutation/types/uhc/BarebonesScenario.java

```diff
@@ -0,0 +1,96 @@
+    }
+
+    // Disable drops for ores that are tiered above iron
+    @EventHandler(priority = EventPriority.HIGH)
+    public void onBlockBreak(BlockBreakEvent event) {
+        Block block = event.getBlock();
+
+        if (block.getType() == Material.REDSTONE_ORE) {
+            event.getItemDrop().remove();
+            block.getState().update();
+            event.getBlock().setType(Material.AIR);
+            player.sendMessage(message("mutation.type.barebones.disabled", ChatColor.RED));
+        }
+
+        if (block.getType() == Material.LAPIS_ORE) {
+            event.getItemDrop().remove();
+            block.getState().update();
+            event.getBlock().setType(Material.AIR);
+            player.sendMessage(message("mutation.type.barebones.disabled", ChatColor.RED));
+        }
+
+        if (block.getType() == Material.EMERALD_ORE) {
+            event.getItemDrop().remove();
+            block.getState().update();
+            event.getBlock().setType(Material.AIR);
+            player.sendMessage(message("mutation.type.barebones.disabled", ChatColor.RED));
+        }
+
+        if (block.getType() == Material.GOLD_ORE) {
+            event.getItemDrop().remove();
+            block.getState().update();
+            event.getBlock().setType(Material.AIR);
+            player.sendMessage(message("mutation.type.barebones.disabled", ChatColor.RED));
+        }
+
+        if (block.getType() == Material.DIAMOND_ORE) {
+            event.getItemDrop().remove();
+            block.getState().update();
+            event.getBlock().setType(Material.AIR);
+            player.sendMessage(message("mutation.type.barebones.disabled", ChatColor.RED));
+        }
+    }
+
+    // Disable anvil crafting
+    @EventHandler(priority = EventPriority.HIGHEST)
+    public void onItemCraft(CraftItemEvent event) {
+        CraftingInventory inventory = event.getInventory();
+
+        if (inventory.getResult() != null && inventory.getResult().getType().equals(Material.ANVIL)) {
+            inventory.setResult(new ItemStack(Material.AIR));
+        }
+    }
+
+    // Disable enchantment table crafting
+    @EventHandler(priority = EventPriority.HIGHEST)
+    public void onItemCraft(CraftItemEvent event) {
+        CraftingInventory inventory = event.getInventory();
+
+        if (inventory.getResult() != null && inventory.getResult().getType().equals(Material.ENCHANTMENT_TABLE)) {
+            inventory.setResult(new ItemStack(Material.AIR));
+        }
+    }
+
+    // Drop certain items on player death
+    @EventHandler(priority = EventPriority.HIGHEST)
+    public void onPlayerDeath(PlayerDeathEvent event) {
+        event.getDrops().add(new ItemStack (Material.DIAMOND, 1));
+        event.getDrops().add(new ItemStack (Material.GOLDEN_APPLE, 1));
+        event.getDrops().add(new ItemStack (Material.STRING, 2));
+        event.getDrops().add(new ItemStack (Material.ARROW, 32));
+    }
+
+    @Override
+    public void disable() {
```

Comment:
No need to override if only calling `super`

Reply:
I mean you can delete the entire `disable()` method since all it does is call the parent method. If the method is not defined by the subclass, the superclass's method will be called instead (the same as using `super`)